src/components/loadingScreen/loadingScreen.css

```diff
@@ -0,0 +1,79 @@
+.loadingscreen-backdrop {
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  position: fixed;
+  background-size:cover;
+  z-index: 4;
+}
+
+.loadingscreen-logo {
+  background-size: cover;
+  z-index: 10;
+  height: 36px;
+  width: 113px;
+  position: absolute;
+  top: 91px;
+  right: 50px;
+}
+
+.loadingscreen-throbber {
+  display: inline-block;
+  position: fixed;
+  top: 0;
+  bottom: 50%;
+  left: 0;
+  right: 0;
+  margin: auto;
+  z-index: 501;
+}
+
+.loadingscreen-text {
+  display: inline-block;
+  position: fixed;
+  color: white;
+  text-align: center;
+  top: 30%;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  width: 45%;
+  height: 15%;
+  margin: auto;
+  z-index: 501;
+}
+
+.loadingscreen-progressbar {
+  display: inline-block;
+  position: fixed;
+  top: 45%;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  width: 50%;
+  margin: auto;
+  z-index: 501;
+  background-color: transparent !important;
```

Comment:
I believe this use of `!important` will be able to removed once added as a `sass` file. CSS specificity should take over and apply the transparency correctly.